src/components/GameRoundFinale/GameRoundFinale.tsx

```diff
@@ -60,17 +62,13 @@ function GameRoundFinale({
           <div className="my-guess-div col-span-2">
             <a
               className="my-guess-p"
-              href={`https://www.google.com/maps?q=${tGeo(potCode)},Canada`}
+              href={`https://www.google.com/maps?q=${tGeo(potCode)},province,Canada`}
```

Comment:
Is `province` needed there? I think it would be clearer for Maps to just search e.g., `YT, Canada` instead of `YT, province, Canada`, especially that YT is not a province but a territory.
Just asking though.

Reply:
nope. yes, that works.
note: when I manually searched for "mb,canada" it found me Mercedes Benz, but with this specific link, MB is ok.
note2: quebec,canada is also took me to a 'wrong' place
Thanks for the point

Reply:
LGTM, thank you! Mercedes-Benz, that's hilarious 🤣